src/Utilities/Math.hpp

```diff
@@ -76,55 +76,85 @@ constexpr T step_function(const T& arg) {
 /*!
  * \ingroup UtilitiesGroup
  * \brief Smoothly interpolates from 0 to 1 between `lower_edge` and
- * `upper_edge` with a Hermite polynomial of degree `2 * N + 1`.
+ * `upper_edge` with Hermite interpolation of polynomial degree `2 * N + 1`.
  *
  * The smoothstep function is
  *
- * \f{align*}
+ * \begin{align*}
  * S_N(x) = \begin{cases}
  * 0 &\quad \text{for} \quad x\leq x_0 \\
  * \tilde{S}_N((x - x_0) / (x_1 - x_0))
  * &\quad \text{for} \quad x_0 \leq x\leq x_1 \\
  * 1 &\quad \text{for} \quad x_1\leq x \\
  * \end{cases}
- * \f}
+ * \end{align*}
  *
- * where \f$x_0\f$ is `lower_edge`, \f$x_1\f$ is `upper_edge`, and, up to
- * \f$N=3\f$,
+ * where \f$x_0\f$ is `lower_edge` and \f$x_1\f$ is `upper_edge`. The general
+ * form of the polynomial \f$\tilde{S}_N(x)\f$ is:
  *
- * \f{align*}
+ * \begin{equation}
+ * \tilde{S}_N(x) = x^{N+1} \sum_{k=0}^{N} \mathrm{binom}(N+k, k)
+ *   \mathrm{binom}(2N+1, N-k) (-x)^k
```

Comment:
I think this doesn't really matter... but in the docs for binomial, you used `\binom{n}{k}` whereas here you write `\mathrm{binom}(n, k)`